iceoryx_posh/test/fuzztests/fuzzing.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2021 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "fuzzing.hpp"
+#include "cpptoml.h"
+#include "iceoryx_utils/platform/socket.hpp"
+
+#include <fstream>
+#include <thread>
+
+std::string const UDS_NAME = "/tmp/";
```

Comment:
The Fuzz Wrappers are only meant for testing purposes and it would be ok for me if for this specific test only Linux is supported since many Fuzzer (which you need additionally to the wrappers) are also only available for Linux. 
Iceoryx itself is not affected because the Fuzz Wrappers shall not run in a productive environment anyhow. 

I will change std::string const UDS_NAME to constexpr char UDS_Name[]